modules/rostests/apitests/gdi32/SetLayout.c

```diff
@@ -0,0 +1,241 @@
+/*
+ * PROJECT:         ReactOS api tests
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Tests for SetLayout and its effects on other gdi functions
+ *                  such as StretchBlt, BitBlt, LPtoDP, DPtoLP
+ * PROGRAMMERS:     Baruch Rutman
+ *                  Inspired by the StretchBlt test
+ */
+
+#include "precomp.h"
+
+static void copy(PUINT32 buffer, UINT32 value, int width, int start_x, int start_y, int end_x, int end_y)
+{
+    for (int y = start_y; y < end_y; y++)
+    {
+        for (int x = start_x; x < end_x; x++)
+            buffer[y * width + x] = value;
+    }
+}
+
+#define BLACK_PIXEL 0x000000
+#define BLUE_PIXEL  0x0000FF
+#define GREEN_PIXEL 0x00FF00
+#define RED_PIXEL   0xFF0000
+#define WHITE_PIXEL 0xFFFFFF
+
+#if 0
+/* Draw the bitmap as colored letters (if console supports color escape codes) */
```

Comment:
With the function `SetConsoleTextAttribute` you can change the colors in a windows console :)

Reply:
Good point, did that and now it works in ReactOS as well.
Thanks for the tip.